lib/tokenizers.js

```diff
@@ -165,11 +181,19 @@ async function _getCurrentTokenizer() {
   return tokenizer;
 }
 
-async function _readCurrentTokenizerRecord() {
+export async function _readCurrentTokenizerRecord({explain = false} = {}) {
   const query = {'tokenizer.current': true};
   const projection = {_id: 0};
   const collection = database.collections['tokenizer-tokenizer'];
-  const record = await collection.findOne(query, projection);
+
+  if(explain) {
+    // 'find().limit(1)' is used here since 'find()' returns a cursor method
```

Comment:
```suggestion
    // 'find().limit(1)' is used here because 'findOne()' doesn't return a cursor method
```

Reply:
And change `cursor method` to just `cursor` -- as I believe that's more accurate, isn't it? (It's not a method that's returned, just a cursor object?)